pkg/crc/machine/machine.go

```diff
@@ -99,6 +116,12 @@ func Start(startConfig StartConfig) (StartResult, error) {
 		result.Status = vmState.String()
 	}
 
+	logging.InfoF(" Waiting 3m0s for the openshift cluster to be started ...")
+	time.Sleep(time.Minute * 3)
```

Comment:
th is a little... well... conclusive and arbitrary ?

Reply:
if no feedback, this is not a good idea...
better to use the so called 'waiting dots again ... OK?'